### ISSUE TITLE
IndentationError error in 3.9.5

### ISSUE THREAD

**Issue by OpenFieldsOfOz** [datawhores/OF-Scraper#377]
## Describe the bug

When running `ofscraper` there is an indentation error

```
$ ofscraper -l DEBUG
Traceback (most recent call last):
  File "/home/user/.local/bin/ofscraper", line 5, in <module>
    from ofscraper.__main__ import main
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/__main__.py", line 4, in <module>
    import ofscraper.runner.load as load
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/runner/load.py", line 6, in <module>
    import ofscraper.runner.run as run
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/runner/run.py", line 5, in <module>
    import ofscraper.commands.picker as picker
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/commands/picker.py", line 1, in <module>
    import ofscraper.commands.check as check
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/commands/check.py", line 12, in <module>
    import ofscraper.api.archive as archived
  File "/home/user/.local/lib/python3.11/site-packages/ofscraper/api/archive.py", line 5
    _______  _______         _______  _______  _______  _______  _______  _______  _______ 
IndentationError: unexpected indent
```

## To Reproduce

Install 3.9.5, run `ofscraper -l DEBUG`

## Expected behavior

ofscraper startup

## System Info

- OS: Ubuntu 22.04
- Browser Chrome
- Version 3.9.5
- python 3.11

**Comment by El-tra** [comment]
Same here.
OS: Win10, x64
Python: 3.12

**Comment by drwitwicki** [comment]
+1

**Comment by mareklibor** [comment]
same here in docker container

**Comment by dodecaphone** [comment]
+1

**Comment by RaySajuuk** [comment]
+3

**Comment by avekifes** [comment]
Sneed.

**Comment by scroguard** [comment]
same here. python 3.11.2, debian 12 lxc container.

**Comment by El-tra** [comment]
Seems to be fixed with 3.9.6!

**Comment by scroguard** [comment]
can confirm, the upgrade to 3.9.6 fixed it for me as well.
